lldb/test/API/lang/swift/external_provider_extra_inhabitants/main.swift

```diff
@@ -0,0 +1,14 @@
+import Cocoa
```

Comment:
this can be `Foundation`

Reply:
Cocoa is an umbrella framework that imports a whole bunch of others. If we can import something more targeted that would be faster.